pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -249,6 +253,19 @@ var (
 	RotationPolicyAlways PrivateKeyRotationPolicy = "Always"
 )
 
+// The type of additional private key format to write
+// +kubebuilder:validation:Enum=DER
+type KeyFormat string
+
+const (
+	// DER Key format is the binary format of the key
+	DERKeyFormat KeyFormat = "DER"
```

Comment:
Perhaps add a note here to mention that this will be stored in a key named `tls.der` :)

Reply:
I've added description of create files according to each format.
I've also added the `PEM` format to create a tls.pem file containing tls.key + tls.crt (concatened)